less/index.less

```diff
@@ -57,3 +79,186 @@ html, body {
     height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+}
+
+body {
+	font-size: 1.6rem;
+}
+
+h1 {
+	font-size: 6.5rem;
+	font-family: @new-font;
+
+	@media (max-width: 500px) {
+			font-size: 3rem;
+	}
+}
+
+h2 {
+	font-size: 3.5rem;
+	font-family: @new-font;
+	padding-bottom: 10px;
+
+	@media (max-width: 500px) {
+		font-size: 2.5rem;		
+	}
+}
+
+h3, a, .intro button {
+	font-size: 2rem;
+}
+
+h3 {
+	font-family: @new-font;
+}
+
+nav {
+	width: 100%;
+	padding: 10px 0;
+	margin-bottom: 30px;
+	display: flex;
+	justify-content: flex-end;
+
+	a {
+		padding-left: 3rem;
+		color: darkgreen;
+	}
+}
+
+.container {
+	max-width: 1000px;
+	width: 100%;
+	margin: 0 auto;
+	padding: 20px 5rem;
+	background-color: lavender;
+
+	@media (max-width: 500px) {
+		min-width: 500px;
```

Comment:
I'm a little baffled by this combination of max-width and min-width... can you elaborate?